tests/flows/native_token_bridge.go

```diff
@@ -56,6 +58,27 @@ func NativeTokenBridge(network interfaces.LocalNetwork) {
 	Expect(err).Should(BeNil())
 	log.Info("Native Token Bridge Contract Address: " + bridgeContractAddress.Hex())
 
+	{
+		// The deployer is used to also send native transfers so we don't have to create an additional address.
+		// The deployer will send 1.25*initialReserveImbalance over the course of the test. Send 2*initialReserveImbalance
+		// native tokens so that it also has enough to cover transaction fees, including deploying the contract.
+		utils.SendNativeTransfer(
+			ctx,
+			sourceSubnet,
+			fundedKey,
+			deployerAddress,
+			utils.BigIntMul(initialReserveImbalance, big.NewInt(2)),
```

Comment:
Can we use the same `sourceFundingAmount` as in `erc20_to_native_token_bridge.go`?

Reply:
No, this test needs more because we need to send native tokens to the dest subnet. The erc20 test doesn't need these because it's sending erc20 tokens

Reply:
Ah yeah, makes sense.